ethan.strader.txt

```diff
@@ -0,0 +1,5 @@
+Hi, I'm Ethan Strader
+Just call me Ethan
+I'm in section 294
+big fan of Ruby on Rails
+I forgot a step
```

Comment:
Awesome job! Please make sure to clean this up on your side to avoid exposing any information to the public you wish to keep private